priv/cabbage/apps/itest/test/itest/in_flight_exits_test.exs

```diff
@@ -62,6 +63,7 @@ defmodule InFlightExitsTests do
   @gas_process_exit_price 1_000_000_000
 
   setup do
+    Reorg.finish_reorg()
     # as we're testing IFEs, queue needs to be empty
     0 = get_next_exit_from_queue()
```

Comment:
I think this is still a problem with eth hash nodes returning revert reasons rather then 0 right?

Reply:
I added a hack for it https://github.com/omgnetwork/elixir-omg/pull/1579/commits/1ef4b25e4bf106cee077dc17e54a8d3b814058f4

Reply:
@InoMurko It seems event with this hack, in-flight tests still fail